test/topology/conftest.py

```diff
@@ -118,9 +118,14 @@ def cluster_con(hosts: List[str], port: int, ssl: bool):
                    # have been more than enough, but in some extreme cases with a very
                    # slow debug build running on a very busy machine, they may not be.
                    # so let's increase them to 60 seconds. See issue #11289.
-                   connect_timeout = 60,
-                   control_connection_timeout = 60,
-                   max_schema_agreement_wait=60,
+                   connect_timeout = 120,
+                   control_connection_timeout = 120,
+                   max_schema_agreement_wait=120,
+                   # The default timeout for heartbeat timeout is 30 seconds.  This should
+                   # have been more than enough, but in some extreme cases with a very
+                   # slow debug build running on a very busy machine, it may not be.
+                   # Increase it to 60 seconds. See issue #11289.
+                   idle_heartbeat_timeout=300,
```

Comment:
The comment mentions 60 but the timeout is 300?
Also, is a value as high as 300 really necessary? Why not 120 like the others?